aspnetcore/razor-pages/rp-mvc.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Razor Pages compared to ASP.NET Core MVC with controllers and views
+author: Rick-Anderson
+description: Contrast and compare Razor Pages to controller/view programming in ASP.NET Core
+monikerRange: '>= aspnetcore-2.0'
+ms.author: riande
+ms.date: 8/05/2018
+uid: razor-pages/rp-mvc
+---
+# Razor Pages compared to ASP.NET Core MVC
+
+By [Scott Sauber](https://twitter.com/scottsauber) and [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+Razor Pages is a feature of the ASP.NET Core MVC framework. Razor Pages follows the MVVM pattern. Developers and teams familiar with ASP.NET MVC development:
+
+* Can continue app development featuring controllers and views.
+* Can be assured the ASP.NET Core framework will provide improvements to controller/view development.
+
+Razor Pages can be added to controller/view projects. The controller-view approach to MVC may be preferred by larger teams.
+
+If you haven't tried Razor Pages, consider using them. Many developers find Razor Pages development simpler and more productive than using controllers and views.
```

Comment:
This doesn't tell me anything either.

Reply:
Customers may find MVC more productive if they're heavily invested in it. Consider an ASP.NET 4.x app that was ported to ASP.NET Core 1.x. The team's devs can follow a pattern that's long been established when creating a new page in the app. My opinion is that it makes ASP.NET Core more approachable to beginners, since there are fewer moving parts. It's easier to ramp up on the tech because of this.